### PR TITLE
chore: Setup loading bar

### LINKED ISSUE #16: Setup loading bar
https://github.com/vercel/next.js/tree/canary/examples/with-loading

### PR DESCRIPTION
**Pull request by mdPial**
Closes #16

### CODE REVIEW

**Comment by mdPial** [comment]
@iamareebjamal Why did you close this without giving any reasons? If I make any mistake I should be informed at least, it will help me to learn otherwise all my efforts are just total waste!!!!!.

**Comment by iamareebjamal** [comment]
Cool. Resolve conflicts and update the PR. You said in an issue that you are unavailable because of exams

**Comment by mdPial** [comment]
> Cool. Resolve conflicts and update the PR. You said in an issue that you are unavailable because of exams

Sorry for the misunderstanding. I meant about working with any new issue.

**Comment by mdPial** [comment]
I will resolve merge conflict and update it :smile:

**Comment by mdPial** [comment]
I have resolved the merge conflict. :smile:

File: pages/_app.tsx [hunk 1]
```diff
@@ -22,6 +32,22 @@ function MyApp({
   }, [])
   return (
     <ThemeProvider theme={theme}>
+      <Head>
+        <link rel="stylesheet" type="text/css" href="/nprogress.css" />
+      </Head>
```
Comment (iamareebjamal) [inline pages/_app.tsx:37]: This should be inside styleprovider

Comment (mdPial) [reply pages/_app.tsx]: I have made suggested changes :smile:

**Pull request merged**